src/stylesheets/components/_banner.scss

```diff
@@ -51,8 +51,8 @@ $banner-icon-close: (
 }
 
 .usa-banner__content {
-  @include grid-container($theme-banner-max-width);
   @include add-responsive-site-margins;
+  @include u-maxw("desktop");
```

Comment:
Shouldn't it be `@include u-maxw($theme-banner-max-width);`?

We have:
```scss
$theme-banner-max-width: "desktop" !default;
```
In `_settings-components.scss`

Reply:
Not in this case, exactly, since valid values for that setting can include widths wider than desktop, or even ‘none’, and the content region needs to be constrained

Reply:
Wouldn't we want to center it on desktop+ to have it align properly to the text above? 

Here's what I see on 1300px
![image](https://user-images.githubusercontent.com/3385219/87056211-ed7a6f00-c1ca-11ea-9a20-a434c639fd71.png)

I know it's a weird issue, but I want to make sure I understand the intention.

Reply:
Ah yes you are right! :facepalm: One sec.

Reply:
I think `@include u-margin-x("auto");` would be enough on `.usa-banner__content`.

I've pushed a commit so you can pull and test.

#### 1300px
![image](https://user-images.githubusercontent.com/3385219/87066310-f6be0880-c1d7-11ea-9b9f-a13069c644b4.png)

Looks the same on smaller sizes too. Let me know if this works or if you intended something else.

Reply:
OK — I have what I think is the proper solution here. We leave everything as it was, but we add a `measure` token to the `usa-banner__guidance` element. `measure-3` seems to be the best option. Pushed the change... take a look.